crates/net/eth-wire/src/types/message.rs

```diff
@@ -153,6 +158,7 @@ pub enum EthMessage {
     NewBlock(Box<NewBlock>),
     Transactions(Transactions),
     NewPooledTransactionHashes(NewPooledTransactionHashes),
+    NewPooledTransactionHashes68(NewPooledTransactionHashes68),
```

Comment:
also need to update struct docs for EthMessage.

Reply:
follow it up with 011eb73e763d5ba6e7ad2f0b3b4cb22d9f58e807. Please suggest if docs is insufficient because I'm not a fluent English speaker.